tests/jointcalTestBase.py

```diff
@@ -255,3 +258,75 @@ def _test_metrics(self, result, expect):
                     self.assertFloatsAlmostEqual(value, expect[key.metric], msg=key.metric, rtol=1e-5)
                 else:
                     self.assertEqual(value, expect[key.metric], msg=key.metric)
+
+    def _importRepository(self, instrument, exportPath, exportFile):
+        """Import a gen3 test repository into self.testDir
+
+        NOTE: this is lifted from Eli's work on DM-24318, and should be moved
+        to a common place, maybe pipe_base?
+
+        Parameters
+        ----------
+        instrument : `str`
+            Full string name for the instrument.
+        exportPath : `str`
+            Path to location of repository to export.
+            This path must contain an `exports.yaml` file containing the
+            description of the exported gen3 repo that will be imported.
+        exportFile : `str`
+            Filename of export data.
+        """
+        self.repo = os.path.join(self.output_dir, 'testrepo')
+
+        # Make the repo and retrieve a writeable Butler
+        _ = lsst.daf.butler.Butler.makeRepo(self.repo)
+        butler = lsst.daf.butler.Butler(self.repo, writeable=True)
+        # Register the instrument
+        instrInstance = lsst.obs.base.utils.getInstrument(instrument)
+        instrInstance.register(butler.registry)
+        # Import the exportFile
+        butler.import_(directory=exportPath, filename=exportFile,
+                       transfer='symlink',
+                       skip_dimensions={'instrument', 'detector', 'physical_filter'})
+
+    def _runPipeline(self, repo, pipelineFile=None, queryString=None,
+                     inputCollections=None, outputCollection=None,
+                     configFiles=None, configOptions=None,
+                     registerDatasetTypes=False):
+        """Run a pipeline via pipetask.
+        """
```

Comment:
This definitely should not be promoted into pipe_base since this is a temporary hack until DM-26239 is done.  We should make a TODO referencing that ticket here.